avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -130,14 +130,18 @@ depMin = 1.0e-6
 curvAcceleration = 1
 
 #++++++++++++Friction model
-# friction type (samosAT, Coulomb)
+# friction type (samosAT, Coulomb, Voellmy)
 frictModel = samosAT
-#+++++++++++++SamosAt friction model
+#+++++++++++++General Friction parameters
+# tan of bed friction angle used for: samosAT, Coulomb, Voellmy
 mu = 0.15500
+#+++++++++++++SamosAt friction model
 Rs0 = 0.222
 kappa = 0.43
 R = 0.05
 B = 4.13
+#+++++++++++++Voellmy friction model
+xsi = 4000.
```

Comment:
General question: should we start adding units in the description?

Reply:
I made an issue for this